packages/back-end/src/api/openapi/payload-schemas/PutExperimentPayload.yaml

```diff
@@ -0,0 +1,154 @@
+type: object
+required:
+  - id
+  - name
+  - variations
+  - owner
+  - datasourceId
+  - trackingKey
+  - exposureQueryId
+properties:
+  datasourceId:
+    type: string
+    description:  ID for the [DataSource](#tag/DataSource_model)
+  exposureQueryId:
+    type: string
+  trackingKey:
+    type: string
+  id:
+    type: string
+    description: ID for the experiment
+  name:
+    type: string
+    description: Name of the experiment
+  project:
+    type: string
+    description: Project ID which the experiment belongs to
+  hypothesis:
+    type: string
+    description: Hypothesis of the experiment
+  description:
+    type: string
+    description: Description of the experiment
+  tags:
+    type: array
+    items:
+      type: string
+  metrics:
+    type: array
+    items:
+      type: string
+  owner:
+    type: string
+    description: Email of the person who owns this experiment
+  archived:
+    type: boolean
+  status:
+    type: string
+    enum:
+      - draft
+      - running
+      - stopped
+  autoRefresh:
+    type: boolean
+  hashAttribute:
+    type: string
+  variations:
+    type: array
+    minItems: 2
+    items:
+      type: object
+      required:
+        - id
+        - key
+        - name
+        - description
+        - screenshots
```

Comment:
These required fields make sense to keep since it is being applied to a nested collection